ZeroMQ socket monitoring

cachedout opened the pull request:
This allows a user to log connection events from a ZeroMQ socket to better diagnose connectivity issues. Requires ZeroMQ 4.x.

jfindlay commented:
@cachedout, how do you feel about putting this into 2015.2 instead?  There are many issues coming from users about (as yet) inexplicable lost minion connectivity and the sooner we can get this into their hands, the better we'll be able to observe and resolve these problems.

cachedout commented:
I have no objection to backporting it. Do you want another PR?

On Tue, Feb 17, 2015 at 2:46 PM, jfindlay <notifications@github.com> wrote:

> @cachedout <https://github.com/cachedout>, how do you feel about putting
> this into 2015.2 instead? There are many issues coming from users about (as
> yet) inexplicable lost minion connectivity and the sooner we can get this
> into their hands, the better we'll be able to observe and resolve these
> problems.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack/salt/pull/20774#issuecomment-74760964>.
>

jfindlay commented:
I would like to have this in 2015.2.  I'm open to however you are willing to do that, thanks.

cachedout commented:
Just go ahead and backport this PR then. If it doesn't patch cleanly let me know and I will do another one.

Commit:
ZeroMQ socket monitoring

This allows a user to log connection events from a ZeroMQ socket to better diagnose connectivity issues. Requires ZeroMQ 4.x.

jfindlay commented:
backport to 2015.2
